net/nut/files/nut-sendmail-notify

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+(cat - | /usr/sbin/sendmail root ) <<EOF
+From: root
+To: root
+Subject: UPS $NOTIFYTYPE Notification
+
+$1
+.
+EOF
```

Comment:
On Mon, 2016-07-04 at 06:14 -0700, Yousong Zhou wrote:
> In net/nut/files/nut-sendmail-notify:
> > @@ -0,0 +1,10 @@
> > +#!/bin/sh
> > +
> > +(cat - | /usr/sbin/sendmail root ) <<EOF
> > +From: root
> > +To: root
> > +Subject: UPS $NOTIFYTYPE Notification
> > +
> > +$1
> > +.
> > +EOF
> Guess there is no need to use subshell. I see no reason why sendmail
> cannot work directly with here document ;)

Erm, yeah.  Do I get a useless use of cat award ;-) ?

Regards,

Daniel